x-pack/plugins/uptime/public/components/settings/add_connector_flyout.tsx

```diff
@@ -30,16 +30,15 @@ export const AddConnectorFlyout = ({ focusInput }: Props) => {
 
   const dispatch = useDispatch();
 
-  useEffect(() => {
-    dispatch(getConnectorsAction.get());
-    focusInput();
-  }, [addFlyoutVisible, dispatch, focusInput]);
-
   const ConnectorAddFlyout = useMemo(
     () =>
       getAddConnectorFlyout({
         consumer: 'uptime',
-        onClose: () => setAddFlyoutVisibility(false),
+        onClose: () => {
+          dispatch(getConnectorsAction.get());
```

Comment:
What does this bit of logic do? Unsure of how to test its move from useEffect to onClose.

Reply:
so when flyout closes, it's refetching all the connector actions, that are displayed in combo box options in settings page.